k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -733,6 +759,35 @@ function deployCortxConfigMap()
         --from-file="${auto_gen_ha_path}"
 }
 
+# https://stackoverflow.com/a/26665585/508923
+function choose()
+{
+    echo "${1:RANDOM%${#1}:1}" "${RANDOM}";
+}
+
+function choosealpha()
+{
+    choose '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+}
+
+function pwgen()
+{
+    local first
+    local rest
+    first=$(choosealpha | awk '{printf "%s", $1}')
+    rest=$({
+        choose '!@#$%^'
+        choose '0123456789'
+        choose 'abcdefghijklmnopqrstuvwxyz'
+        choose 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+        for i in $( seq 1 $(( 4 + RANDOM % 8 )) )
+        do
+            choosealpha
+        done
+    } | sort -R | awk '{printf "%s", $1}')
```

Comment:
One of the comments in that SO link says that `-R` is a GNU extension. Are we sure `sort -R` works on something like MacOS?

Reply:
I didn't realize that.  I have replaced this with another technique.